packages/ui/form/PhoneInput.tsx

```diff
@@ -59,17 +58,13 @@ function BasePhoneInput({ name, className = "", onChange, ...rest }: PhoneInputP
 
 const useDefaultCountry = () => {
   const [defaultCountry, setDefaultCountry] = useState("us");
```

Comment:
We keep the initial state "us" if the country is not supported we keep displaying us when the page is loaded(as demonstrated on loom video)